k8s/bases/swe-d2cc/swe-d2cc-adapter.yaml

```diff
@@ -36,6 +36,13 @@ data:
     swe-runner:
       binding:
         destination: swe-d2cc-request
+        
+    minio-adapter:
+      bucket: ${MINIO_BUCKET}
```

Comment:
You should also add these parameters in the env block in the deployment part to set valueFrom